solutions/devsprint-bruno-faganello-2/SpaceApp/SpCustomBtn.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SpCustomBtn.swift
+//  SpaceApp
+//
+//  Created by Tatiana Rico on 26/04/22.
+//
+
+import SwiftUI
+
+struct SpCustomBtn: View {
```

Comment:
[REQUEST CHANGED]
Change name to SPButtonView

Reply:
corrigido

Reply:
Aqui pra mim ainda ta com o nome antigo e está faltando passar um title pro Button pra deixar ele o mais genérico possível